content/articles/2017-11-21-usethis-1-0-0.Rmd

```diff
@@ -0,0 +1,144 @@
+date: '2017-11-21'
+slug: usethis-1.0.0
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/IClZBVw5W5A
+  author: Todd Quackenbush
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're pleased to announce the release of [usethis 1.0.0](https://github.com/r-lib/usethis/releases/tag/v1.0.0) 
+on CRAN (now [1.1.0](https://github.com/r-lib/usethis/releases/tag/v1.1.0), following some small improvements discovered after release). usethis is designed to smooth package development workflows by automating everything that can be automated. Many of the functions were previously part of devtools, but have been extracted out in their own package to give them more room to grow.
+
+```{r, eval = FALSE}
+install.packages("usethis")
+```
+
+## R packages
+
+Get started by loading usethis and running `create_package()`. For this blog post I'm going to create a package in a temporary directory, but normally you'd create it somewhere safe!
+
+```{r}
+library(usethis)
+
+# Create a new package -------------------------------------------------
+tmp <- file.path(tempdir(), "mypkg")
+create_package(tmp)
+```
+
+If you're using RStudio, this will automatically open a new RStudio session. Generally, usethis is tightly integrated with RStudio but should work with other editors too. The fallbacks for non-RStudio editors aren't as well tested, so please [let us know](https://github.com/r-lib/usethis/issues) if you encounter problems.
+
+(If you find yourself using usethis a lot and don't want to load it everytime, run `use_usethis()` and follow the instructions.)
+
+Notice that the output includes "Changing active project to mypkg". usethis has the concept of an "active project" which is where all usethis functions will have effect. It is usually the working directory, but is changed automatically by the `create_`  functions, and can be changed manually with `proj_set()`.
+
+If you don't want a package, use `create_project()` to create a minimal [RStudio project](https://support.rstudio.com/hc/en-us/articles/200526207-Using-Projects). usethis is gradually evolving towards supporting more general R "projects" but this is still a work in progress. Please [let us know](https://github.com/r-lib/usethis/issues) if you have ideas or discover a problem!
+
+### Code and tests
+
+`use_r()` will create/edit a script file in `R/`:
+
+```{r}
+use_r("foo")
+```
+
+`use_test("foo")` will set up all the infrastructure you need for [unit testing](http://r-pkgs.had.co.nz/tests.html), and create/edit a test file in `tests/teststhat/`:
+
+```{r}
+use_test("foo")
+```
+
+We recommend that you have paired test and source files. If you use RStudio, you can use `use_test()` to create/edit the test file corresponding to the open source file, and `use_r()` to create/edit to the source file corresponding to the open test file. 
+
+### Dependencies
+
+To declare that your project has some dependencies, use `use_package()`:
```

Comment:
"project" in this case really does mean "package"

Reply:
Welllllllllllll, a project could have dependencies, and I think it would make sense to declare them in the DESCRIPTION.